src/BallInf.jl

```diff
@@ -134,46 +142,57 @@ It is defined as the norm of the enclosing ball, of the given
 ### Output
 
 A real number representing the norm.
+
+### Notes
+
+The norm of an infinity ball is defined as the norm of the enclosing ball, of
+the given ``p``-norm, of minimal volume.
 """
 function norm(B::BallInf, p::Real=Inf)::Real
     return maximum(map(x -> norm(x, p), vertices_list(B)))
 end
 
 """
-    radius(B::BallInf, [p])
+    radius(B::BallInf, [p]::Real=Inf)
 
 Return the radius of a ball in the infinity norm.
 
-It is defined as the radius of the enclosing ball of the given ``p``-norm of
-minimal volume with the same center.
-
 ### Input
 
-- `B` -- a ball in the infinity norm
+- `B` -- ball in the infinity norm
 - `p` -- (optional, default: `Inf`) norm
 
 ### Output
 
 A real number representing the radius.
+
+### Notes
+
+The radius is defined as the radius of the enclosing ball of the given
```

Comment:
we can put the definition as the second sentence on top, as in some other cases

Reply:
I was sticking to the [guidelines](https://github.com/JuliaReach/LazySets.jl/wiki/Documentation-Guidelines#writing-docstrings-) which say "One-line purpose of the function". I am open for changing it.